docs/deploy-cephfs.md

```diff
@@ -44,7 +44,7 @@ that should be resolved in v14.2.3.
 **Available command line arguments:**
 
 | Option                    | Default value               | Description                                                                                                                                                                                                                                                                          |
-|---------------------------|-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ------------------------- | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
@humblec how these changes are made(it is via a plugin)?  how to catch this issues? i would like to have a check or update our CI to catch this one.